cmd/install.go

```diff
@@ -118,15 +120,10 @@ func setupSubCommand(config *configs.Config,
 		subCommand.SetActionFunc(func(ctx context.Context, args []string) error {
 			ctx, err := configureLogging(ctx, commandMessage, config)
 			if err != nil {
+				//Logger doesn't exist
 				return err
```

Comment:
prefer to wrap error with context to let user easy to tracking.
like:
  `return errors.Wrap(err, "failed to configure logging option")`